db/schema.rb

```diff
@@ -590,7 +590,11 @@
     t.string "email_language", limit: 10
     t.datetime "accepted_terms_at", precision: nil
     t.datetime "encrypted_recovery_code_digest_generated_at", precision: nil
+    t.datetime "suspended_at"
+    t.datetime "reinstated_at"
+    t.index ["reinstated_at"], name: "index_users_on_reinstated_at"
```

Comment:
Are these indexes used for a specific query?

Reply:
It is highly probable that it will use.

Reply:
Can you share a bit more on that? If it's primarily going to be used for reporting or outside of the request cycle, it may be worth exploring other options.

In general, we should avoid an index if we can, or scope it to the extent that it will be needed.

Reply:
I agree, I don't think we'll need to list all suspended users on a frequent basis, mostly just spot-checking individual users, right?

Reply:
If we'll only be spot-checking individual users, it's probably fine to drop the indexes? Can always re-add them later if needed.

Reply:
Yes! individual users. Agree! I will drop this.